Scheduler: add Overview (#6625)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/02 Overview/00 Overview.md

> DevExtreme Scheduler is a UI component for scheduling that implements flexible data binding, easy appointment editing, multiple calendar views, time zones support, and more.

Reviewer: ```suggestion
DevExtreme Scheduler is a UI scheduling component used to implement flexible data binding, easy appointment editing, multiple calendar views, time zone support, and more.
```

> In this overview you can find a map of Scheduler elements, information about key features, and what to explore next.

Reviewer: ```suggestion
In this overview, you can find a map of Scheduler elements, information about key features, and what to explore next.
```
